src/components/Request/link/views/Create.request.link.view.tsx

```
@@ -241,8 +244,23 @@ export const CreateRequestLinkView = () => {
+    // update the sharebutton section to remove the loading state logic:
+    {
+        isCreatingLink ? (
+            <Button disabled={true} shadowSize="4">
+                <div className="flex w-full flex-row items-center justify-center gap-2">
+                    <Loading /> {' Loading'}
+                </div>
+            </Button>
+        ) : (
+            <ShareButton generateUrl={generateLink}>Share Link</ShareButton>
+        )
+    }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)